core/runtime/src/main/java/io/quarkiverse/jberet/runtime/QuarkusJobOperator.java

```diff
@@ -33,17 +34,20 @@
     private final BatchEnvironment batchEnvironment;
     private final Map<String, Job> jobs;
     private final JBeretConfig config;
+    private final ThreadPoolConfig threadPoolConfig;
 
     public QuarkusJobOperator(
             final JBeretConfig config,
             final ManagedExecutor managedExecutor,
             final TransactionManager transactionManager,
-            final List<Job> jobs) {
+            final List<Job> jobs,
+            final ThreadPoolConfig threadPoolConfig) {
 
-        this.batchEnvironment = new QuarkusBatchEnvironment(config, new QuarkusJobExecutor(managedExecutor, config),
+        this.batchEnvironment = new QuarkusBatchEnvironment(config, new QuarkusJobExecutor(managedExecutor, threadPoolConfig),
                 transactionManager);
         this.jobs = jobs.stream().collect(Collectors.toMap(Job::getJobXmlName, job -> job));
         this.config = config;
+        this.threadPoolConfig = threadPoolConfig;
```

Comment:
I think we don't need to store this in a field here since we pass it directly to the `QuarkusJobExecutor`.

Reply:
Removed !!